src/main/java/bio/terra/janitor/service/cleanup/StairwayLoggingHooks.java

```diff
@@ -0,0 +1,74 @@
+package bio.terra.janitor.service.cleanup;
+
+import bio.terra.janitor.db.TrackedResourceId;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.HookAction;
+import bio.terra.stairway.StairwayHook;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+public class StairwayLoggingHooks implements StairwayHook {
+  private static final String FlightLogFormat =
+      "Operation: {}, flightClass: {}, flightId: {}, timestamp: {}";
+  private static final String StepLogFormat =
+      "Operation: {}, flightClass: {}, flightId: {}, stepIndex: {}," + "timestamp: {}";
+  private static final String TRACKED_RESOURCE_MDC_KEY = "trackedResourceId";
+  private static final Logger logger = LoggerFactory.getLogger(StairwayHook.class);
+
+  @Override
+  public HookAction startFlight(FlightContext context) {
+    logger.info(
+        FlightLogFormat,
+        "startFlight",
+        context.getFlightClassName(),
+        context.getFlightId(),
+        Instant.now().atZone(ZoneId.of("Z")).format(DateTimeFormatter.ISO_INSTANT));
+    return HookAction.CONTINUE;
+  }
+
+  @Override
+  public HookAction startStep(FlightContext context) {
+    logger.info(
+        StepLogFormat,
+        "startStep",
+        context.getFlightClassName(),
+        context.getFlightId(),
+        context.getStepIndex(),
+        Instant.now().atZone(ZoneId.of("Z")).format(DateTimeFormatter.ISO_INSTANT));
```

Comment:
Stairway does its own logging that each step starts end. This isn't adding much that we can use since it's not even including the tracked resource id. I'm not sure this is helpful to add, and if we do want it, let's do it after the mdc scope is set.

Reply:
This is the pattern I saw in data repo:
https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/java/bio/terra/service/job/StairwayLoggingHooks.java
I thought it might be good to add log flight with more context. So I copied the whole thing over